jme3-core/src/main/java/com/jme3/renderer/opengl/GL3.java

```diff
@@ -41,7 +41,9 @@
 public interface GL3 extends GL2 {
 
     public static final int GL_DEPTH_STENCIL_ATTACHMENT = 0x821A;
+    public static final int GL_DST_ALPHA = 0x0304;
```

Comment:
These two constants are available in both OpenGL 1.1 and OpenGL ES 2.0, so they should go into the `GL` interface